### PR TITLE
Release candidate changes

### PR DESCRIPTION
**Pull request by shamantraghav**
As the release gets triggered from CLI-core, I don't see an input field that changes the formula.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
     inputs:
       formula:
         description: 'Formula to release Homebrew'
-        default: twilio
+        default: twilio@rc
```
Comment (onuzbee) [inline .github/workflows/release.yml:7]: Does it not pass on the inputs from twilio-cli?

Comment (shamantraghav) [reply .github/workflows/release.yml:7]: It does not even ask for this in cli-core. We'll need to changed that

Comment (onuzbee) [reply .github/workflows/release.yml:7]: [This](https://github.com/twilio/twilio-cli/blob/9bd5494264600ace8bd27752a1da59fecf53eda9/.github/workflows/oclif-release.yml#L69) line

Comment (onuzbee) [reply .github/workflows/release.yml:7]: Oh, got that. Thanks! Approved.

**Review by onuzbee — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (shamantraghav) [reply .github/workflows/release.yml:7]: I Will change it after the release cause I don't want to spend time on testing the whole pipeline now

**Pull request merged**